.vagrant.d/data/checkpoint_signature

```diff
@@ -0,0 +1,6 @@
+8d0487ef-cbde-4671-a09e-5f3af00b5f1d
+
+This signature is a randomly generated UUID used to de-duplicate
+alerts and version information. This signature is random, it is
+not based on any personally identifiable information. To create
+a new signature, you can simply delete this file at any time.
```

Comment:
Strange, I have a ```.vagrant``` folder, not a ```.vagrant.d``` and only ```.vagrant``` is in the ```.gitignore```. Can you remove these files from your commit? I'll make a PR to add ```.vagrant.d``` to the ```.gitignore``` for you.